### PR TITLE
PUB-1037 - Lists Manual Removal

### PR DESCRIPTION
**Pull request by commitBlob**
### JIRA link ###
https://tools.hmcts.net/jira/browse/PUB-1037

### Change description ###
Add removal flow building from the `remove-list-search` page (PUB-1036)

**NOTE:**
Delete does not work at the moment as this ticket depends on PUB-1038.
PR for that is raised: https://github.com/hmcts/pip-data-management/pull/65

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```
list-summary
removepulbication

remove-success

### CODE REVIEW

File: src/test/unit/views/remove-list-confirmation.test.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      });
+    });
+
+    it('should display error summary', () => {
+      const dialog = htmlRes.getElementsByClassName('govuk-error-summary');
+      expect(dialog[0].getElementsByClassName('govuk-error-summary__title')[0].innerHTML)
+        .contains('There is a problem', 'Could not find error dialog title');
+    });
+
+    it('should display error messages in th the summary', () => {
```
Comment (kurt-uk) [inline src/test/unit/views/remove-list-confirmation.test.ts:53]:
```suggestion
    it('should display error messages in the summary', () => {
```

**Review by kurt-uk — CHANGES_REQUESTED** [review]

**Review by kurt-uk — APPROVED** [review]
nice work!

**Review by furnivall — APPROVED** [review]
looks good 👍

**Review by furnivall — DISMISSED** [review]
👍

**Review by kurt-uk — APPROVED** [review]

**Review by alberto-hortiguela — DISMISSED** [review]

**Review by ChrisS1512 — APPROVED** [review]

**Pull request merged**